lumeer-core/src/main/java/io/lumeer/core/facade/OrganizationFacade.java

```diff
@@ -146,4 +140,20 @@ public void removeGroupPermission(final String organizationCode, final String gr
       organization.getPermissions().removeGroupPermission(group);
       organizationDao.updateOrganization(organization.getId(), organization);
    }
+
+   private void createOrganizationScopedRepositories(Organization organization) {
+      projectDao.setOrganization(organization);
+      projectDao.createProjectsRepository(organization);
+
```

Comment:
Do we really need to setOrganization before creating its repository? Same with delete..

Reply:
Yes, because projects are stored in a separate collection in the user database now. And I have used organization ID in the mongo collection name so system and user database can be the same (as we use it for testing).